lib/web_purifex/parser.ex

```diff
@@ -31,7 +31,11 @@ defmodule WebPurifex.Parser do
       |> Kernel.||("0")
       |> String.to_integer
 
-    {:ok, %Response{status: status, found: found}}
+    expletive =
+      response_body
+      |> get_in(["rsp", "expletive"])
```

Comment:
Would always returning a list (empty or not) make the usage of the library easier instead of `nil | binary | list` ?

Reply:
:-) I had considered that but decided to stay as close to the API. I'm on the fence about changing the original API. Thoughts?

Reply:
This is just an opinion (/2cents):
- an API should be as predictable as possible, so returning multiple types is a mistake to me
- we are doing an abstraction anyway by building this library, we can leverage that to fix the smells

:)

Reply:
Well, since we both considered this, probably suggests returning a single type is a good way to go. Will update. 👍